lib/subscription.rb

```diff
@@ -25,9 +25,22 @@ def self.get_subscriptions(services)
     end.flatten
   end
 
+  def self.send_webhook_notification(webhook_url, claim_review)
+    RestClient.post(webhook_url, {claim_review: claim_review}.to_json)
+  end
+
+  def self.safe_send_webhook_notification(webhook_url, claim_review, raise_error=true)
+    begin
+      self.send_webhook_notification(webhook_url, claim_review)
```

Comment:
Will it always raise an exception when the response is not 200? For example, if the webhook URL is correct but the response is a 400 or 500 (sorry I'm not very familiar with `RestClient`).

Reply:
Yes, it will always raise an exception for any non-200 response.

Reply:
(AFAIK)